docs/quality.md

```diff
@@ -1,3 +1,60 @@
 # Quality
 
-> @wip
+- Don't want to break this for driver, mongoose
+- Want input from these maintainers on establishing best practies, not to just
+  create more work
+- Result of this will be incorporated into our tooling,
+  [mj](https://github.com/rueckstiess/mj), for automated organization wide
+  checking and scaffolding for new projects
+
+## Testing
+
+- `npm test` returns a non-zero exit code
+- travisci hook
+
+## Code Coverage
+
+- do it if possible
+- tooling will template out istanbul + mocha + coveralls.io
+
+## Documentation
+
+- A lot of different needs here
+- For general small modules, examples in README + comments where it makes sense
+- If you expose more than `module.exports = function(){}`, use jsdoc tags + `dox --api`
+- Tooling will publish a pretty version to github pages for you
+
+## Style
+
+- A `.jshintrc` [example](https://github.com/imlucas/khaos-amp/blob/master/template/.jshintrc)
+- A `.jsfmtrc` [example](https://github.com/imlucas/khaos-amp/blob/master/template/.jsfmtrc)
```

Comment:
MMS is setup to use jscs as well -- though we haven't agreed on our preferred rules.  How well does the jsfmt re-writing work?  (The MMS Go agent repos really like gofmt.)